bareos/bconsole.sls

```diff
@@ -18,6 +18,10 @@ install_bconsole_package:
     - require:
       - pkgrepo: bareos_repo
     {% endif %}
+    {% if salt['pillar.get']('bareos:generate_unique_password', False) %}
```

Comment:
@arthurlogilab, how about adding this default in `defaults.yaml`

```yaml
bareos:
  generate_unique_password: false
```

And just do

```yaml
{% if bareos.generate_unique_password %}
```

wdyt?

Also, I'm not sure if a `require_in` needs this check first, but anyway :smile:

Reply:
@javierbertoli I agree that this syntax is more elegant. I'll submit a change in that direction next time I work on this project. 

For `require_in` you mean that if the state does not exist it just ignores it ?

Reply:
> For require_in you mean that if the state does not exist it just ignores it ?

Exactly. But again, I'm not completely sure if that's the case.